bullet_train/docs/billing/stripe.md

```diff
@@ -42,96 +89,209 @@ Note this is different than how many Rails engines ask you to install migrations
 
 This "products" list is what Stripe will display to users as upgrade and downgrade options in the customer portal. You shouldn't list any products here that aren't properly configured in your Rails app, otherwise the resulting webhook will fail to process. If you want to stop offering a plan, you should remove it from this list as well.
 
-## 8. Finalize Webhooks Testing by Managing a Subscription
+### 8. Finalize Webhooks Testing by Managing a Subscription
 
 In the same account where you created your first test subscription, go into the "billing" menu and click "manage" on that subscription. This will take you to the Stripe Billing customer portal.
 
 Once you're in the customer portal, you should test upgrading, downgrading, and canceling your subscription and clicking "⬅ Return to {Your Application Name}" in between each step to ensure that each change you're making is properly reflected in your Bullet Train application. This will let you know that webhooks are being properly delivered and processed and all the products in both systems are properly mapped in both directions.
 
+#### 8.1 Understanding the Product configuration yml
+
+Your `products.yml` should have a structure similar to the following:
+
+```yaml
+free:
+  limits:
+    memberships:
+      count: 1
+      enforcement: hard
+    scaffolding/absolutely_abstract/creative_concepts:
+      count: 0
+      enforcement: hard
+basic:
+  price: basic_2021
+  image: "streamline-icon-idea-message.png"
+  limits:
+    memberships:
+      count: 3
+      enforcement: hard
+    scaffolding/absolutely_abstract/creative_concepts:
+      count: null
+  prices:
+    basic_2021:
+      amount: 1000
+      currency: USD
+      duration: 1
+      interval: month
+      quantity: memberships
+    basic_2021:
+      amount: 10000
+      currency: USD
+      duration: 1
+      interval: month
+      quantity: memberships
+```
+
+Each root level entry (i.e. `free` and `basic`) are the types of subscriptions that will be available to your customers. Each subscription has the following:
+
+  - `price`: The product price key **discuss during PR to get a better understanding of what this means here**
+  - `image`: The product image from `assets/images/products` that is presented to the buyer from the new subscription page
+  - `limits`: The resource constraints on what a user can do on this subscription. To learn more, refer to [Bullet Train Usage Limits](/docs/billing/usage.md)
+  - `prices`: The list of pricing options for this product, where each price will have the following:
+    - `amount`: The cost of the product in *cents*
+    - `currency`: The currency that the customer will be billed in (i.e. EUR for Euros, CAD for Canadian Dollars)
+    - `duration`: Used for subscriptions to define how many units the subscription is valid for
+    - `interval`: The unit of time of the subscription (i.e. day, month, year)
```

Comment:
We might mention that `duration` and `interval` are related. For instance if you set `duration: 2` and `interval: month` then people will be billed every other month. If you set `duration: 3` and `interval: month` then they'll be billed every third month.